src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,93 @@
+package duke.command;
+
+import duke.exception.DukeException;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class Duke {
+    // Constants
+    public static final String LOGO = " ____        _        \n"
+                                    + "|  _ \\ _   _| | _____ \n"
+                                    + "| | | | | | | |/ / _ \\\n"
+                                    + "| |_| | |_| |   <  __/\n"
+                                    + "|____/ \\__,_|_|\\_\\___|\n";
+    public static final String MESSAGE_BOUNDARY = "____________________________________________________________";
+    public static final String FILE_NAME = "data/duke.txt";
+
+    // Attributes
+    private static final TaskOperation doTask = new TaskOperation();
+
+    // Main function
+    public static void main(String[] args) throws DukeException, FileNotFoundException {
+        Greet();
+        doTask.loadFile(FILE_NAME);
+        Echo();
+        doTask.saveFile(FILE_NAME);
+
+    }
+
+    public static void Greet(){
+        System.out.println("Hello from\n" + LOGO);
+        String greet = " Hello! I'm duke.command.Duke\n"
+                    + " What can I do for you?\n";
+        System.out.println(MESSAGE_BOUNDARY + "\n" + greet + MESSAGE_BOUNDARY);
+    }
+
+    public static void Echo() throws DukeException {
+        boolean isChat;
+        Scanner in = new Scanner(System.in);
+
+        do{
+            // Get query from user
+            String userInput = in.nextLine();
+
+            // Give response
```

Comment:
It's a bit unclear to say "Give response". It's better if you can specify what kind of response